rotkehlchen/api/v1/schemas.py

```diff
@@ -3031,3 +3044,18 @@ class EthStakingHistoryStatsDetails(EthStakingHistoryStats, AsyncIgnoreCacheQuer
 
 class SkippedExternalEventsExportSchema(Schema):
     filepath = FileField(required=True, allowed_extensions=['.csv'])
+
+
+class ExportHistoryEventSchema(HistoryEventSchema):
+    """Schema for quering history events"""
+    directory_path = DirectoryField(required=True)
+
+    def make_extra_filtering_arguments(self, data: dict[str, Any]) -> dict[str, Any]:
+        return {}
+
+    def validation_proxied_information(self, data: dict[str, Any]) -> dict[str, Any]:
+        extra_fields = {}
+        directory_path = data.get('directory_path', None)
+        if directory_path is not None:
+            extra_fields['directory_path'] = directory_path
```

Comment:
`.get` returns `None` by default when not found

```suggestion
        if (directory_path := data.get('directory_path')) is not None:
            extra_fields['directory_path'] = directory_path
```

Reply:
thanks just noticed the operator too ':='